lib/needle.js

```diff
@@ -478,6 +486,11 @@ Needle.prototype.send_request = function(count, method, uri, config, post_data,
     done(err || new Error('Unknown error when making request.'));
   }
 
+  function abort_handler() {
+    out.emit('err', new Error('Aborted by signal.'));
+    request.abort();
```

Comment:
request.abort() is deprecated. You should use request.destroy() instead.

<img width="1249" alt="image" src="https://user-images.githubusercontent.com/80770179/203999478-3da69289-ebe5-43b3-81ba-8e78eac5b3c8.png">

Reply:
Good catch. Done, thanks.